### ISSUE TITLE
Vex.Flow.Voice.getBoundingBox() returns too large rectangle with ClefNote (or some others?)

### ISSUE THREAD

**Issue by sug1no** [0xfe/vexflow#343]
please refer to the following test:

~~http://jsfiddle.net/184sthee/~~
http://jsfiddle.net/184sthee/57/

![image](https://user-images.githubusercontent.com/3293067/45333070-28032880-b5af-11e8-97b4-9700607ff346.png)

```
voiceTop:  Object { x: 0, y: 0, w: 73.44064, h: 295 }
StaveNote: Object { x: 61.422399999999996, y: 255, w: 12.01824, h: 40 }
ClefNote:  Object { x: 0, y: 0, w: 0, h: 0 }
TimeSigNoteNote:  Object { x: 0, y: 0, w: 0, h: 0 }
BarNote:  Object { x: 0, y: 0, w: 0, h: 0 }
```
---

Since {BarNote, ClefNote, TimeSigNote}.getBoundingBox() returns (0, 0, 0, 0)
merged rectangle size is too large.

https://github.com/0xfe/vexflow/blob/master/src/clefnote.js#L44
https://github.com/0xfe/vexflow/blob/master/src/voice.js#L210

is this a bug or limitation?

**Comment by 0xfe** [comment]
Looks like `ClefNote.getBoundingBox()` is not implemented, so calling it dispatches `Note.getBoundingBox()` which returns `null`. This should be an easy fix.

**Comment by sug1no** [comment]
fixed by #651.
